Sources/BraintreePaymentFlow/BTLocalPaymentResult.swift

```diff
@@ -0,0 +1,57 @@
+@objcMembers public class BTLocalPaymentResult: BTPaymentFlowResult {
+    
+    /// The billing address.
+    public let billingAddress: BTPostalAddress
```

Comment:
Are all these fields non-optional?

Reply:
These are non-optional, yes. `BTLocalPaymentResult` is instantiated internally (see TODO below in file).